GameStore.Services/Services/Implementation/GameServices.cs

```diff
@@ -27,6 +27,14 @@ public ICollection<GameModel> GetAll()
             return games;
         }
 
+        public ICollection<GameInfoTransferModel> GetGameInfoTransferAll()
+        {
+            var games = GameStoreMapper.Map<ICollection<Game>, ICollection<GameInfoTransferModel>>
```

Comment:
It will look better if you will have something like this:
`var games = gameRepository.GetAll();
return Mapper.....`

Reply:
Fixed